### PR TITLE
Add steps for setting up kubectl context for licensify

### PR DESCRIPTION
**Pull request by theseanything**
This is needed for accessing the clusters for the first time.

### CODE REVIEW

File: source/manual/access-to-licensing-for-third-parties.html.md [hunk 1]
```diff
@@ -70,12 +70,34 @@ If you can't use the GDS CLI, you can use the `aws-vault exec` command with your
 aws-vault exec govuk-integration -- aws sts get-caller-identity
 ```
 
+### Accessing the cluster for the first time
+
+1. Set your AWS credentials as enviroment variables:
```
Comment (ChrisBAshton) [inline source/manual/access-to-licensing-for-third-parties.html.md:75]:
```suggestion
1. Set your AWS credentials as environment variables:
```

File: source/manual/access-to-licensing-for-third-parties.html.md [hunk 2]
```diff
@@ -70,12 +70,34 @@ If you can't use the GDS CLI, you can use the `aws-vault exec` command with your
...
+    ```
+
 Once you're authenticated with AWS, you can check your connection to Kubernetes:
 
 ```sh
-gds aws govuk-integration-licensinguser --  kubectl cluster-info
+kubectl -n licensify get deploy
 ```
 
+You will need to do this setup for each environment. Make sure to replace references to "integation" in the command above with the relevant environment name.
```
Comment (ChrisBAshton) [inline source/manual/access-to-licensing-for-third-parties.html.md:99]:
```suggestion
You will need to do this setup for each environment. Make sure to replace references to "integration" in the command above with the relevant environment name.
```

**Review by ChrisBAshton — APPROVED** [review]
⭐

**Comment by theseanything** [comment]
@ChrisBAshton ahah thank you! (Note to self: get spellcheck)

**Pull request merged**